models/DataType.yaml

```diff
@@ -0,0 +1,5 @@
+keyword: String # Keyword that identifies the type i.e. int, str
+# Integer attributes
+integer-min: String # String representing minium integer or uncertain
```

Comment:
The available types are at https://github.com/thanethomson/statik/wiki/Models , and you must use Int if the yaml will contain values which are numerical according to the yaml spec

Reply:
I did it like that since statik only supports unsigend 32 bit integer.

Reply:
@jayvdb ^

Reply:
or change them to strings.